Directory.Packages.props

```diff
@@ -0,0 +1,18 @@
+<Project>
+  <PropertyGroup>
+    <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageVersion Include="EasyBuild.FileSystemProvider" Version="0.3.0" />
+    <PackageVersion Include="Fixie.TestAdapter" Version="3.4.0" />
+    <PackageVersion Include="FSharp.Core" Version="7.0.300" />
+    <PackageVersion Include="FsToolkit.ErrorHandling" Version="4.16.0" />
+    <PackageVersion Include="Ionide.KeepAChangelog.Tasks" Version="1.0.0-test-002" />
+    <PackageVersion Include="KeepAChangelogParser" Version="1.2.4" />
+    <PackageVersion Include="Microsoft.Build.Utilities.Core" Version="17.11.4" />
```

Comment:
Bumping the MSbuild version to 17.11 means that only VS 17.11 (aka .NET SDK 8.0.4xx) can use this task. We need to consider what our desired support is - I'd be ok with all .NET 8 versions only, which would mean using the 17.8 version of MSbuild packages.

Reply:
I would possibly have gone with .NET 6 SDK as it's still in support, but if you're happy with .NET 8 only it lets us use the latest `Dotnet.Reproduciblebuilds` which is generally tidier